src/browser/provider/built-in/index.js

```diff
@@ -0,0 +1,14 @@
+import PathBrowserProvider from './path';
+import LocalBrowserProvider from './local';
+import RemoteBrowserProvider from './remote';
+import saucelabsBrowserProvider from './saucelabs';
+import phantomJSBrowserProvider from './phantomjs';
+
+
+export default {
+    'local':     new LocalBrowserProvider(),
+    'path':      new PathBrowserProvider(),
```

Comment:
The `path` provider designed to take a path to the browser executable, figure out required command line arguments and start it. So yeah, it's less useful if the browser is properly installed and it is a common one (chrome, firefox, opera) - in this case you can use `locally-installed` provider and specify just browser name. 

But turns out that the `locally-installed` provider cannot detect exotic browsers yet, so the only way to start such browser is through the `path` provider.